apps/src/code-studio/components/progress/UnitOverview.jsx

```diff
@@ -55,6 +55,7 @@ class UnitOverview extends React.Component {
     isCsdOrCsp: PropTypes.bool,
     completedLessonNumber: PropTypes.string,
     isProfessionalLearningCourse: PropTypes.bool,
+    publishedState: PropTypes.string,
```

Comment:
Similar to how we scoped the PropTypes to be oneOf in `UnitOverviewTopRow`, let's pull that up here too.